microsoft/testsuites/ltp/ltp.py

```diff
@@ -87,14 +93,18 @@ def run_test(
         block_device: Optional[str] = None,
         temp_dir: str = "/tmp/",
         ltp_run_timeout: int = 12000,
+        # run full test if True, else run lite test
+        run_full_test: bool = False,
     ) -> List[LtpResult]:
-        # tests cannot be empty
-        assert_that(ltp_tests, "ltp_tests cannot be empty").is_not_empty()
+        if not run_full_test:
+            # ltp_tests cannot be empty
+            assert_that(ltp_tests, "ltp_tests cannot be empty").is_not_empty()
```

Comment:
How about empty ltp_tests means run all? So it doesn't need the `fun_full_test` variable.